gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -429,6 +448,14 @@ def run(argv=None):
 
   beam_pipeline_options = pipeline_options.PipelineOptions(pipeline_args)
   pipeline = beam.Pipeline(options=beam_pipeline_options)
+  google_cloud_options = beam_pipeline_options.view_as(
+      pipeline_options.GoogleCloudOptions)
+  file_path_to_file_hash = pipeline_common.create_file_path_to_file_hash_map(
```

Comment:
This could be moved in either the if statement below, or even into the _create_sample_info_table method directly.